tests/test_models.py

```diff
@@ -61,7 +61,6 @@ def test_serialize_a_recommendation(self):
         self.assertIn("rec_product_id", data)
         self.assertEqual(data["rec_product_id"], recommendation.rec_product_id)
         self.assertIn("type", data)
-        self.assertEqual(data["type"], recommendation.type.name)
```

Comment:
Is there a reason to delete this line?

Reply:
I should have removed it by mistake will add that line